deploy/olm-catalog/argocd-operator/0.5.0/argocd-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -756,91 +756,102 @@ spec:
       statusDescriptors:
       - description: 'ApplicationController is a simple, high-level summary of where
           the Argo CD application controller component is in its lifecycle. There
-          are five possible ApplicationController values: Pending: The Argo CD application
+          are four possible ApplicationController values: Pending: The Argo CD application
```

Comment:
IMPORTANT:

This file should not have any changes. 

PLease do not merge this before we update the Version Make Target to `0.6.0` and regenerate the manifests.